pkg/bootstrap/bootstrap_config.go

```diff
@@ -112,6 +112,22 @@ func RunProxy(config *meshconfig.ProxyConfig, node string, epoch int, configFnam
 	return cmd.Process, nil
 }
 
+// GetHostPort separates out the host and port portions of an address. The
+// host portion may be a name, IPv4 address, or IPv6 address (with square brackets).
+func GetHostPort(name, addr string) (host string, port string, err error) {
```

Comment:
Not sure this and next method need to be exported. If you think other pieces will use it - maybe move to a different package, but probably not worth it.

Reply:
Yeah, I could make this private. Should I do that now, or is it too late?